src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctRobot = {};
+
+  for (const key in robot) {
+    if (correctRobot.hasOwnProperty([robot[key]])) {
+      return null;
+    }
+    correctRobot[robot[key]] = key;
```

Comment:
the checklist, p.1

Reply:
I added a variable, but I don't understand is it right or not (